__tests__/html2/speech/js/MockedSpeechSynthesisUtterance.js

```diff
@@ -0,0 +1,122 @@
+import { EventTargetProperties } from 'https://unpkg.com/event-target-properties@latest/dist/event-target-properties.mjs';
+
+export default class SpeechSynthesisUtterance extends EventTarget {
+  constructor(text) {
+    super();
+
+    this.#eventTargetProperties = new EventTargetProperties(this);
+    this.#text = text || '';
+  }
+
+  #eventTargetProperties;
+  #lang;
+  #pitch;
+  #rate;
+  #text;
+  #voice;
+  #volume;
+
+  get lang() {
+    return this.#lang;
+  }
+
+  set lang(value) {
+    this.#lang = value;
+  }
+
+  get pitch() {
+    return this.#pitch;
+  }
+
+  set pitch(value) {
+    this.#pitch = value;
+  }
+
+  get rate() {
+    return this.#rate;
+  }
+
+  set rate(value) {
+    this.#rate = value;
+  }
+
+  get text() {
```

Comment:
Just curious, why these need to be get/set if they can just be public fields?

Reply:
It's just better mock. :)

But agree it could be just fields.